src/cmd/services/m3query/config/config.go

```diff
@@ -120,6 +122,9 @@ type Configuration struct {
 
 	// LookbackDuration determines the lookback duration for queries
 	LookbackDuration *time.Duration `yaml:"lookbackDuration"`
+
+	// Cache configurations.
+	Cache CacheConfigurations `yaml:"cache"`
```

Comment:
Added this instead of directly adding query conversion cache in case we have more cache options to add in the future